runtime/worker.rs

```diff
@@ -78,7 +78,7 @@ pub fn validate_import_attributes_callback(
   for (key, value) in attributes {
     let msg = if key != "type" {
       Some(format!("\"{key}\" attribute is not supported."))
-    } else if value != "json" {
+    } else if value != "json" && value != "$$deno-core-internal-wasm-module" {
```

Comment:
I probably should have exported that string in `deno_core` or handled the validation there.